### PR TITLE
Pmidc ginni finance pi 8206

### PR DESCRIPTION
**Pull request by egovmis1**

### CODE REVIEW

File: egov/egov-egf/src/main/resources/reports/templates/bankAdviceReport.jrxml [hunk 1]
```diff
@@ -1,58 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```
Comment (subhash-eGov) [inline egov/egov-egf/src/main/resources/reports/templates/bankAdviceReport.jrxml:2]: Add the copy right which is removed.

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/report/BankAdviceReportAction.java [hunk 2]
```diff
@@ -467,11 +473,13 @@ public String exportPDF() {
         final Map<String, Object> reportParams = new HashMap<String, Object>();
         final StringBuffer letterContext = new StringBuffer();
         letterContext
-                .append("             I request you to transfer the amount indicated below through RTGS duly debiting from the")
+                .append("             I request you to transfer the amount indicated below through RTGS/NEFT duly debiting from the")
                 .append("  Current Account No: ")
                 .append(getBankAccountNumber(bankaccount.getId()) != null ? getBankAccountNumber(bankaccount.getId()) : " ")
                 .append("  under your bank to the following bank accounts:");
         reportParams.put("bankName", getBankName(bank.getId()));
+        final City city = cityService.getCityByCode(ApplicationThreadLocals.getCityCode());
```
Comment (subhash-eGov) [inline egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/report/BankAdviceReportAction.java:481]: City Name also present in ApplicationThreadLocals. Read from Thread locals if it is present.

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java [hunk 3]
```diff
@@ -160,6 +160,7 @@
     private PersistenceService persistenceService;
     @Autowired
     private EgovCommon egovCommon;
+    private CityService cityService;
```
Comment (subhash-eGov) [inline egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java:163]: Remove this if not used.

**Review by subhash-eGov — CHANGES_REQUESTED** [review]

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/report/BankAdviceReportAction.java (hunk 2 above)
Comment (egovmis1) [reply egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/report/BankAdviceReportAction.java:481]: i am reading from ApplicationThreadLocals.getCityCode() only

File: egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java (hunk 3 above)
Comment (egovmis1) [reply egov/egov-egfweb/src/main/java/org/egov/egf/web/actions/voucher/PreApprovedVoucherAction.java]: done

File: egov/egov-egf/src/main/resources/reports/templates/bankAdviceReport.jrxml (hunk 1 above)
Comment (egovmis1) [reply egov/egov-egf/src/main/resources/reports/templates/bankAdviceReport.jrxml:2]: added.kindly check once

**Review by shwethadc-eGov — APPROVED** [review]

**Review by shwethadc-eGov — APPROVED** [review]

**Review by shwethadc-eGov — APPROVED** [review]

**Review by shwethadc-eGov — APPROVED** [review]

**Review by shwethadc-eGov — APPROVED** [review]

**Review by subhash-eGov — APPROVED** [review]

**Pull request merged**